app/code/Magento/WishlistGraphQl/etc/schema.graphqls

```diff
@@ -23,7 +23,8 @@ type WishlistOutput @doc(description: "Deprecated: `Wishlist` type should be use
 }
 
 type Wishlist {
-    id: ID @doc(description: "The unique ID for a `Wishlist` object")
+    id: ID @deprecated(reason: "Use `uid` instead") @doc(description: "The unique ID for a `Wishlist` object")
+    uid: ID @doc(description: "The unique ID for a `Wishlist` object")
```

Comment:
@shikhamis11 Good work on the PR! I see you have introduced uid for wishlist, but it will complicate things at this point as it will welcome conflicts on existing mutations. @mauragcyrus We need to have a follow up story, if needed. @shikhamis11 Can you remove this uid here and on the wishlist item. If there are any concerns please reply. thanks!

Reply:
@prabhuram93  thanks for review. I added `uid ` for `Wishlist` and `WishlistItemInterface` as in `addWishlistItemsToCart` mutation we require `wishlistUid` & `wishlistItemUids`

Reply:
yeah agree. I think that's confusing. I will work with @cpartica on renaming that to wishlistId & wishlistItemItds.